mmv1/products/workstations/api.yaml

```diff
@@ -0,0 +1,189 @@
+    base_url: "projects/{{project}}/locations/{{location}}/workstationClusters"
+    create_url: "projects/{{project}}/locations/{{location}}/workstationClusters?workstationClusterId={{workStationCluster_id}}"
+    update_verb: :PATCH
+    update_mask: true
+    min_version: beta
+    description: "A managed workstation cluster."
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        "Workstations": "https://cloud.google.com/workstations/docs/"
+      api: "https://cloud.google.com/workstations/docs/reference/rest/v1beta/projects.locations.workstationClusters/create"
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: "name"
+        base_url: "{{op_id}}"
+        wait_ms: 1000
+        timeouts: !ruby/object:Api::Timeouts
+          insert_minutes: 10
+          update_minutes: 10
+          delete_minutes: 10
+      result: !ruby/object:Api::OpAsync::Result
+        path: "response"
+      status: !ruby/object:Api::OpAsync::Status
+        path: "done"
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: "error"
+        message: "message"
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: "workstationClusterId"
+        required: true
+        input: true
+        url_param_only: true
+        description: |
+          The ID of the workstation cluster.
+      - !ruby/object:Api::Type::String
+        name: "location"
+        input: true
+        url_param_only: true
+        description: |
+          The location where the workstation cluster should reside.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: "name"
+        output: true
+        description: |
+          The name of the cluster resource.
+      - !ruby/object:Api::Type::String
+        name: "uid"
+        output: true
+        description: |
+          The system-generated UID of the resource.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: "labels"
+        description: "Client-specified labels that are applied to the resource and that are also propagated to the underlying Compute Engine resources."
+      - !ruby/object:Api::Type::String
+        name: "network"
+        required: true
+        input: true
+        description: |
+          The relative resource name of the VPC network on which the instance can be accessed. 
+          It is specified in the following form: "projects/{projectNumber}/global/networks/{network_id}".
+      - !ruby/object:Api::Type::String
+        name: "subnetwork"
+        required: true
+        input: true
+        description: |
+          Name of the Compute Engine subnetwork in which instances associated with this cluster will be created. 
+          Must be part of the subnetwork specified for this cluster.
+      - !ruby/object:Api::Type::String
+        name: "displayName"
```

Comment:
could you flex this field in your tests?

Reply:
I've set `display_name` in the `TestAccorkstationsWorkstationCluster_update` test.